spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -1697,6 +1697,17 @@
           end
         end
       end
+
+      describe 'pluck' do
+        (PolicyMachine::POLICY_ELEMENT_TYPES).each do |type|
+          it "can get data for #{type}" do
+            id = "#{type}-pluck-test"
+            policy_machine.send("create_#{type}", id)
+            data = policy_machine.send(:pluck, type: type, fields: [:unique_identifier], options: { unique_identifier: id })
+            expect(data[0]).to eq(id)
```

Comment:
I would actually test the entire data structure like
```suggestion
            expect(data).to eq([id])
```

Reply:
updated